### ISSUE TITLE
508-defect-1 [TESTING]: COVID Vaccine Trials - Form must have e2e tests for all views

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18225]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As an accessibility specialist, I want every form page to have end to end tests that surface hidden content and run axe checks.

## Details

The review and submit page of the COVID vaccine trial form doesn't have any end to end tests. These tests with automated axe checks are required for our ATO, so I'd like to expand on the existing test set. Linked file and code snippet below.

## Acceptance Criteria

- [ ] All existing e2e tests pass
- [ ] New e2e test is added
- [ ] New test opens the accordion(s) on the review page before running an axe check
- [ ] Axe checks return 0 violations

## Environment

* https://staging.va.gov/coronavirus-research/volunteer/review-and-submit

## Proposed Solution (if known)

The previous tests are collected in this file: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/coronavirus-research/tests/00.covid-vaccine-trial.e2e.spec.js

```diff
module.exports = E2eHelpers.createE2eTest(client => {
  client
    .url(`${E2eHelpers.baseUrl}/covid-research/volunteer`)
    .waitForElementVisible('body', Timeouts.normal)
    .axeCheck('.main');

+ client
+   .url(`${E2eHelpers.baseUrl}/covid-research/volunteer/review-and-submit`)
+   .waitForElementVisible('body', Timeouts.normal)
+   .click('button.usa-button-unstyled')
+   .axeCheck('.main');

  client.end();
});
```

**Comment by 1Copenut** [comment]
Verified the e2e test coverage is 💯 and has axe checks at the right times. Moving this issue to close.
